app/uk/gov/hmrc/eusubsidycompliancefrontend/models/json/digital/package.scala

```diff
@@ -43,19 +42,20 @@ package object digital {
       val lead: BusinessEntity =
         o.undertakingBusinessEntity match {
           case h :: Nil => h
-          case _ => throw new IllegalStateException(s"unable to create undertaking with missing or multiple business entities")
+          case _        =>
```

Comment:
Like other alignment remarks, I think this was fine as is. No need to align the arrows. We can keep everything on one line in this instance.

Reply:
I like all arrows alignment  in one line :) I think it depends on person to person.  May be i should hold off this